lib/linters/attribute_quotes.js

```diff
@@ -1,46 +1,35 @@
 'use strict';
 
-module.exports = function (options) {
-    var config = options.config;
-    var node = options.node;
-    var selector;
-    var value;
-
-    // Bail if the linter isn't wanted
-    if (!config.attributeQuotes || (config.attributeQuotes && !config.attributeQuotes.enabled)) {
-        return null;
-    }
+module.exports = {
```

Comment:
I hate to be this nitpicky but the empty lines after each of the `module.exports` and function definitions shouldn't be there, for consistency with other parts of the code. `"disallowPaddingNewlinesInBlocks": true` in `.jshintrc` can help catch some of them.